src/components/home.js

```diff
@@ -33,18 +33,39 @@ const HomePage = () => {
     <>
       <main>
         <section>
-          <h2>Books</h2>
-          <ul>
+          <ul className="bookList">
             {books.map((book) => (
               <li key={book.item_id}>
-                <h3>{book.title}</h3>
-                <p>{book.author}</p>
-                <button
-                  type="button"
-                  onClick={() => handelRemove(book.item_id)}
-                >
-                  Remove
-                </button>
+                <div className="bookInfo">
+                  <div>
+                    <h3>{book.title}</h3>
+                    <p>{book.author}</p>
+                  </div>
+                  <div className="btns">
+                    <button type="button">Comment</button>
+                    <button
+                      type="button"
+                      onClick={() => handelRemove(book.item_id)}
+                    >
+                      Remove
+                    </button>
+                    <button type="button">Edit</button>
+                  </div>
+                </div>
+                <div className="bookProgress">
```

Comment:
- [ ] Good job, your design looks great 👏 But there are a few things you need, please add the missing default category on the book page is missing as you can see on the screenshot below ⬇️  Also category link uses a different color. Please fix them in order to make your project design great 👍 

![Bookstore CMS (1)](https://user-images.githubusercontent.com/6615394/230447048-879be562-1241-40de-9a16-efe229fa7ea7.jpg)

Reply:
#### Thanks @codecaiine for your feedback